bin/index.js

```diff
@@ -5,12 +5,12 @@ const { existsSync, readFileSync } = require('fs');
 const {
   generateRequestFile,
   generateAirtableFile,
-  generateSchemaFile
```

Comment:
ah.. I actually don't know where my global Prettier settings are...... there's even a `prettier.config.js` in this repo now 😂 😭